doc/source/reference/random/c-api.rst

```diff
@@ -108,7 +122,7 @@ It provides low-level functions for various distributions. All the functions req
 
 .. c:function:: double random_rayleigh(bitgen_t *bitgen_state, double mode)
 
-.. c:function:: double random_standard_t(bitgen_t *bitgen_state, double df)
+.. c:function:: double random_student_t(bitgen_t *bitgen_state, double df)
 
 .. c:function:: double random_noncentral_chisquare(bitgen_t *bitgen_state, double df,
```

Comment:
this probably needs to be on a single line as well; the rendering of this line and a couple of others is off, see https://10421-908607-gh.circle-artifacts.com/0/home/circleci/repo/doc/build/html/reference/random/c-api.html

Reply:
fixing

Reply:
looks good now